contents/docs/self-host/deploy/digital-ocean.mdx

```diff
@@ -107,9 +132,15 @@ helm install -f values.yaml --timeout 20m --create-namespace --namespace posthog
 
 ### 3. Lookup external IP
 
-```console
-kubectl get svc --namespace posthog posthog-ingress-nginx-controller
-```
+You can lookup the external IP address in two ways:
+
+1. Navigate to the IP address of the load balancer created by Kubernetes using your web browser. You can grab this by using the DigitalOcean web console ([networking tab](https://cloud.digitalocean.com/networking/load_balancers)).
+2. If you have `kubectl` installed and configured for the cluster that PostHog is deployed to then run these commands to get your installation location:
+    ```console
+    export INGRESS_IP=$(kubectl get --namespace posthog ingress posthog -o jsonpath="{.status.loadBalancer.ingress[0].ip}")
```

Comment:
Also, does this need to be repeated or should there just be one section for obtaining IP? Thinking about what might happen if we update this command in one place but not the other

Reply:
I think the value is that it gives the reader the information where they need it rather than having to go off to another section. The content is quite small - not a big guide - so seems okay to duplicate inline.

In one version I'd moved the command over to a component/include/snippet to reduce duplication and to allow for editing in one place. But there were a few different versions. Maybe I'll do an in-page snippet rather than creating a new file for each type.

Reply:
I've moved the IP address command into a .mdx included snippet in https://github.com/PostHog/posthog.com/pull/1826/commits/6b5042e0fb3eaa7d58e9e05f6d38122a45e05173